values.yaml

```diff
@@ -7,8 +7,18 @@
 ## If you are not sure, find it like this: `grep cluster-cidr /etc/kubernetes/manifests/kube-controller-manager.yaml | cut -d"=" -f2`
 ## PS: if you had flannel before, your clusters will have "10.244.0.0/16"
 
+podCIDR: ""
 #podCIDR: "192.168.0.0/16"
 typha: false
 flexVolumePluginDir: "/usr/libexec/kubernetes/kubelet-plugins/volume/exec"
 vethMtu: 1440
 autoHostEndpoints: false
+
+typha_env:
+  # TYPHA_PROMETHEUSMETRICSPORT: "9093"
```

Comment:
Can we make it consistent with node settings and have `typha.env` please? I like having `node` and `typha` top level keys, as they can be then extended in the future.

Reply:
Yes, but this is breaking, since `typha` is already a boolean.

Reply:
Right. Hmm, I think it should be fine to change it. I don't expect there is many users of this chart right now and the sooner that happens, the better.

```yaml
typha:
  enable: false
  env:
```

With that, let's bump the chart version to `0.3.0`.

Reply:
Good choice! I love it.